opm/simulators/flow/FlowGenericProblem_impl.hpp

```diff
@@ -131,24 +129,14 @@ helpPreamble(int,
 
 template<class GridView, class FluidSystem, class Scalar>
 std::string
-EclGenericProblem<GridView,FluidSystem,Scalar>::
+FlowGenericProblem<GridView,FluidSystem,Scalar>::
 briefDescription()
 {
-    if (briefDescription_.empty())
```

Comment:
why this? Any possible implications?

Reply:
This was the banner used by the 'ebos' binary. As there is no ebos binary...